### PR TITLE
Add captive prompts for the rest of MFA methods

### PR DESCRIPTION
**Pull request by nikosdion**
Pull Request for comment https://github.com/joomla/joomla-cms/pull/37912#issuecomment-1141972256.

Ping @Kostelano @richard67 

### Summary of Changes

Adds captive login prompts for these MFA methods:
* Backup Codes
* TOTP (Authenticator Code)
* YubiKey

The rest of the methods already had it:
* Email
* Fixed
* WebAuthn

### Testing Instructions

* Set up MFA using authentication code and YubiKey
* Note down your Backup Codes
* Log out and log back in

### Actual result BEFORE applying this Pull Request

Choosing to validate with backup codes, TOTP or YubiKey would only display a text box without any help text to help you understand what to do.

### Expected result AFTER applying this Pull Request

There is help text above the text box explaining how to obtain the code and proceed with the MFA.

### Documentation Changes Required

None

### Adds language strings

These three methods didn't have a captive prompt language string. I added three strings.

### CODE REVIEW

File: administrator/language/en-GB/com_users.ini [hunk 1]
```diff
@@ -392,6 +392,7 @@ COM_USERS_USERS_TABLE_CAPTION="Table of Users"
 COM_USERS_USER_ACCOUNT_DETAILS="Account Details"
 COM_USERS_USER_BACKUPCODE="Backup Code"
 COM_USERS_USER_BACKUPCODES="Backup Codes"
+COM_USERS_USER_BACKUPCODES_CAPTIVE_PROMPT="If you do not have access to your usual Multi-factor Authentication method use any of your Backup Codes in the field below. Please remember that this emergency backup code will become invalid after successfully logging in and cannot be reused."
```
Comment (brianteeman) [inline administrator/language/en-GB/com_users.ini]:
```suggestion
COM_USERS_USER_BACKUPCODES_CAPTIVE_PROMPT="If you do not have access to your usual Multi-factor Authentication method use any of your Backup Codes in the field below. Please remember that this emergency backup code cannot be reused."
```

File: language/en-GB/com_users.ini [hunk 2]
```diff
@@ -140,6 +140,7 @@ COM_USERS_RESET_REQUEST_LABEL="Please enter the email address for your account.
 COM_USERS_SETTINGS_FIELDSET_LABEL="Basic Settings"
 COM_USERS_USER_BACKUPCODE="Backup Code"
 COM_USERS_USER_BACKUPCODES="Backup Codes"
+COM_USERS_USER_BACKUPCODES_CAPTIVE_PROMPT="If you do not have access to your usual Multi-factor Authentication method use any of your Backup Codes in the field below. Please remember that this emergency backup code will become invalid after successfully logging in and cannot be reused."
```
Comment (brianteeman) [inline language/en-GB/com_users.ini]:
```suggestion
COM_USERS_USER_BACKUPCODES_CAPTIVE_PROMPT="If you do not have access to your usual Multi-factor Authentication method use any of your Backup Codes in the field below. Please remember that this emergency backup code cannot be reused."
```

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on b5c7d4aade459e9a828efba8d452a7b957fe9434<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38082">issues.joomla.org/tracker/joomla-cms/38082</a>.</sub>

**Comment by Kostelano** [comment]
I have tested this item :white_check_mark: successfully on b5c7d4aade459e9a828efba8d452a7b957fe9434<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38082">issues.joomla.org/tracker/joomla-cms/38082</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38082">issues.joomla.org/tracker/joomla-cms/38082</a>.</sub>

**Comment by fancyFranci** [comment]
Thank you for the improvement!

**Pull request merged**